core/shared/src/main/scala/sttp/client/FollowRedirectsBackend.scala

```diff
@@ -74,6 +82,7 @@ class FollowRedirectsBackend[F[_], S, WS_HANDLER[_]](delegate: SttpBackend[F, S,
   }
 
   private val contentHeaders = Set(HeaderNames.ContentLength, HeaderNames.ContentType, HeaderNames.ContentMd5)
+  private val sensitiveHeaders = Set(HeaderNames.Authorization, HeaderNames.Cookie, HeaderNames.SetCookie)
```

Comment:
maybe these could be a parameter of the class, with a default value as above?

Reply:
I also though about it but was not sure. The fact that you are mentioning it convinces me.